app/src/main/java/explorer/city/com/cityexplorer/util/ImageUtils.kt

```diff
@@ -0,0 +1,11 @@
+package explorer.city.com.cityexplorer.util
+
+import android.content.Context
+import android.widget.ImageView
+import com.bumptech.glide.Glide
+
+fun ImageView.load(context: Context, url: String, view: ImageView) {
```

Comment:
You don't need to pass in `Context` or `ImageView`. This is an extension function, you are extending `ImageView`. `this` points to the instance of the class.

You can therefore reference `this.context` and `this`; simplifying this function to having `url` as its only parameter.